docs/pages/faq-errors.rst

```diff
@@ -13,4 +13,17 @@ When connecting to a remote node, the following error can occur:
     Couldn't connect to peer, reconnecting in XYZ seconds (peer=some.domain.nl:5555,err=unable to connect: context deadline exceeded: connection closed before server preface received)
 
 This indicates the server is using TLS, but the local node is trying to connect without TLS.
-Check the `network.tls.enabled` setting.
\ No newline at end of file
+Check the `network.tls.enabled` setting.
+
+Error: `JWT signing key not present on this node`
+*************************************************
+
+When inspecting an access token, the following error can occur:
+
+    Error while inspecting access token (error: JWT signing key not present on this node)
+
+This indicates the JWT token you send to the Nuts node can't be introspected by the Nuts node,
+because it can't find the private key to create the token. This can have 2 causes:
+
+# Calls are mixed up; token introspection must be done by the Nuts node (*authorizer*) that created it for the consuming node (*requestor*).
```

Comment:
> Calls are mixed up;

Do you mean that the call went to the wrong node? (So nodes are mixed up?)

Reply:
For JWT tokens the node creates its own keypair on startup, it's not DID based